com.usebruno.Bruno.metainfo.xml

```diff
@@ -9,7 +9,7 @@
   <url type="bugtracker">https://github.com/usebruno/bruno/issues</url>
 
   <metadata_license>CC0-1.0</metadata_license>
-  <project_license>MIT</project_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Hello @helloanoop 
Why did you change the licence here? As far as I can see, bruno (https://github.com/usebruno/bruno?tab=MIT-1-ov-file#readme) still has a MIT licence.

I am worried because now we see a "proprietary" badge on the flathub
![image](https://github.com/flathub/com.usebruno.Bruno/assets/43372966/79c06241-35e8-496e-88d2-2bfbc4654107)

jfyi: https://github.com/usebruno/bruno/issues/2112

Reply:
Not sure if this should be split out into a separate issue for better tracking but I believe that a more appropriate `project_license` tag would be `<project_license>MIT AND LicenseRef-proprietary=https://example.org/legal/</project_license>` (I could not find where the actual proprietary license was located to update the link to, so that would need adjustment once identified) due to it [allowing expression operators](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#license) such as [AND](https://spdx.github.io/spdx-spec/v2-draft/SPDX-license-expressions/#d43-conjunctive-and-operator). This would conform to the packages description stating
> Majority of our features are free and have been opensourced under the MIT license
whilst also hopefully making the license badge more accurate.